Gemfile.lock

```diff
@@ -109,19 +109,19 @@ GEM
     rack (1.5.5)
     rack-test (0.6.3)
       rack (>= 1.0)
-    rails (4.1.0)
-      actionmailer (= 4.1.0)
-      actionpack (= 4.1.0)
-      actionview (= 4.1.0)
-      activemodel (= 4.1.0)
-      activerecord (= 4.1.0)
-      activesupport (= 4.1.0)
+    rails (4.1.1)
```

Comment:
ERROR: [Medium][Cross Site Scripting] Rails 4.1.1 does not encode JSON keys (CVE-2015-3226). Upgrade to Rails version 4.1.11
https://groups.google.com/d/msg/rubyonrails-security/7VlB_pck3hU/3QZrGIaQW6cJ